docs/source/settings.rst

```diff
@@ -234,7 +234,13 @@ keepalive
 
 The number of seconds to wait for requests on a Keep-Alive connection.
 
-Generally set in the 1-5 seconds range.
+Generally set in the 1-5 seconds range for servers with direct connection
```

Comment:
Ah, it gets copied from there... alright, thanks for the guidance... I adjusted PR accordingly.